composer.json

```diff
@@ -12,6 +12,7 @@
         }
     ],
     "require": {
-        "twig/twig": "^3.6"
+        "twig/twig": "^3.6",
+        "symfony/asset": "^6.3"
```

Comment:
i think you can remove this line and run composer update or composer remove symfony/asset